server/mailet/mailetcontainer-api/src/main/java/org/apache/james/mailetcontainer/api/MailetLoader.java

```diff
@@ -22,6 +22,8 @@
 import org.apache.mailet.Mailet;
 import org.apache.mailet.MailetConfig;
 
+import java.util.List;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.List'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.17.0.2:9000/coding_rules#rule_key=squid%3AUselessImportCheck)